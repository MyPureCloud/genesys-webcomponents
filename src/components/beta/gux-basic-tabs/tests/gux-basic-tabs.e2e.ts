import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';

describe('gux-basic-tabs', () => {
  it('renders', async () => {
    const html = `
      <gux-basic-tabs id="interactive" lang="en">
        <span slot="tabs">
          <gux-basic-tab tab-id="1" tab-icon-name="lock">
            <span slot="title"> Hello World </span>
          </gux-tab>

          <gux-tab tab-id="2" tab-icon-name="lock">
            <span slot="title"> Hello World 2 </span>
          </gux-tab>
        </span>
      </gux-tabs>
    `;
    const page = await newE2EPage({ html });
    const element = await page.find('gux-basic-tabs');

    expect(element.innerHTML).toMatchSnapshot();
  });
});
