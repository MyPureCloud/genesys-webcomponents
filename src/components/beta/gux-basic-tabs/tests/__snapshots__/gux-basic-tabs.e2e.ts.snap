// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gux-basic-tabs renders 1`] = `" <span slot=\\"tabs\\"> <gux-basic-tab tab-id=\\"1\\" tab-icon-name=\\"lock\\" class=\\"hydrated\\"><!----> <gux-tab tab-id=\\"2\\" tab-icon-name=\\"lock\\" hidden=\\"\\" class=\\"hydrated\\"><!----> <button type=\\"button\\" class=\\"gux-tab gux-show\\" role=\\"button\\"><div class=\\"tab-icon-container\\"><gux-icon icon-name=\\"lock\\" class=\\"hydrated\\"><div class=\\"gux-icon-container\\"><svg xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 16 16\\" aria-hidden=\\"true\\"><path d=\\"M12.5 7H12V5c0-1.763-1.166-3.244-2.758-3.77l-.005-.001a3.826 3.826 0 0 0-.573-.149c-.216-.041-.436-.067-.664-.067s-.448.026-.664.067a3.826 3.826 0 0 0-.578.151C5.166 1.756 4 3.237 4 5v2h-.5C2.671 7 2 7.67 2 8.5v6A1.5 1.5 0 0 0 3.5 16h9a1.5 1.5 0 0 0 1.5-1.5v-6A1.5 1.5 0 0 0 12.5 7zM6 5a2 2 0 0 1 4 0v2H6V5z\\"></path></svg></div></gux-icon></div><span class=\\"tab-title\\"><span slot=\\"title\\">Hello 2</span></span></button></gux-tab> <button class=\\"gux-basic-tab\\" type=\\"button\\" role=\\"button\\"><span class=\\"tab-title\\"><span slot=\\"title\\">Hello</span></span></button></gux-basic-tab></span> "`;
